neurolib/models/wc/timeIntegration.py

```diff
@@ -71,43 +71,42 @@ def timeIntegration(params):
     max_global_delay = np.max(Dmat_ndt)
     startind = int(max_global_delay + 1)  # timestep to start integration at
 
-    x_ou = params["x_ou"]
-    y_ou = params["y_ou"]
+    e_ou = params["e_ou"]
+    i_ou = params["i_ou"]
 
-    x_ext = params["x_ext"]
-    y_ext = params["y_ext"]
+    e_ext = params["e_ext"]
+    i_ext = params["i_ext"]
 
     # set of the state variable arrays
-    xs = np.zeros((N, len(t)))
-    ys = np.zeros((N, len(t)))
+    est = np.zeros((N, len(t)))
+    ist = np.zeros((N, len(t)))
```

Comment:
Is there a particular reason why you didn't call these `es` and `is`?

Reply:
Yes, `is` is a reserved keyword in Python.

Reply:
Of course... damn.

Reply:
What about `e_s` or `i_s`? Is a bit ugly.

Reply:
`exc` and `inh` might be an option too.

Reply:
I agree `e_s` and `i_s` are a bit ugly. My thought was, since these are the 'states', that I would call them `est` and `ist`. I am also happy with `exc` and `inh`.

Reply:
To be honest, the reason why I called them `xs` before was that I wanted to say "it's a few x's" ☺️ ... I suggest we go for `exc` and `inh`? That should look familiar for users and actually fits the WC model. What do you think?

Reply:
Ok, will change now